wordpress/wp-content/plugins/memberful-wp/src/content_filter.php

```diff
@@ -13,6 +13,9 @@ function memberful_wp_protect_content( $content ) {
     // Disable Beaver Builder
     remove_action( "the_content", "FLBuilder::render_content" );
 
+    // Remove Elementor action hook
+    remove_action("elementor/frontent/the_content", "memberful_wp_protect_content");
```

Comment:
Is this a typo?

```suggestion
    remove_action("elementor/frontend/the_content", "memberful_wp_protect_content");
```

Reply:
I'm not sure if I'm following here, but is this even needed if it worked without it (e.g. with a typo)?

Reply:
Thanks, yeah that was a typo. I didn't manage to reproduce the issue myself, but apparently without removing the action there is a risk of an infinite loop: https://github.com/elementor/elementor/issues/8930#issuecomment-525884269